staging-web/config.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import os
+from pathlib import Path
+
+# Application Configuration
+APP_TITLE = "KernelCI Staging Control"
+APP_VERSION = "2.0"
+HOST = "0.0.0.0"
+PORT = 9090
+
+# Database Configuration
+DATABASE_URL = "sqlite:///./staging.db"
+
+# Authentication Configuration
+SECRET_KEY = "kernelci-staging-secret-key-change-in-production"
+ALGORITHM = "HS256"
+ACCESS_TOKEN_EXPIRE_MINUTES = 30
+
+# Paths Configuration
+BASE_PATH = Path("/home/kernelci/kernelci-deploy")
+STAGING_WEB_PATH = BASE_PATH / "staging-web"
+KERNEL_SCRIPT_PATH = BASE_PATH / "kernel.py"
+TREE_FILE_PATH = BASE_PATH / ".tree"
+
+# KernelCI Deployment Paths
+API_PATH = BASE_PATH / "checkout" / "kernelci-api"
+PIPELINE_PATH = BASE_PATH / "checkout" / "kernelci-pipeline"
+
+# SSH Key Configuration
+SSH_KEY_PATH = BASE_PATH / "keys" / "id_rsa_staging.kernelci.org"
+
+# Docker Compose Configuration
+COMPOSE_FILES = [
+    "-f", "docker-compose.yaml",
+    "-f", "docker-compose-kcidb.yaml", 
+    "-f", "docker-compose-lava.yaml",
+    "-f", "docker-compose-production.yaml"
+]
+
+API_SERVICES = ["api", "db", "redis"]
+
+# GitHub Configuration
+GITHUB_REPO = "kernelci/kernelci-core"
+GITHUB_WORKFLOW = "staging.yml"
+GITHUB_REF = "main"
+
+# Kernel Tree Configuration
+KERNEL_TREES = {
```

Comment:
what about kernel_tree that dosen't have a staging branch?

Reply:
Staging is being tested by default on 3 trees derived from mainline, we copy them to github/kernelci/linux and artificially we add commit so trigger process will start building this specific tree.
Others can be triggered by kci-dev.